src/Navigation/Navigation/NavigationServiceOptionsBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CodeMonkeys.Navigation
+{
+    public class NavigationServiceOptionsBase
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the <see cref="CodeMonkeys.Navigation.INavigationService" />
+        /// should cache built content instances.
+        /// This is required if pages should be prebuilt (<see cref="CodeMonkeys.Navigation.INavigationService.Register{TViewModel}(Type, bool)" />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 147 characters long line (which is greater than 120 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)